mapviz_plugins/CMakeLists.txt

```diff
@@ -120,6 +157,7 @@ install(TARGETS ${PROJECT_NAME}
   ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
 )
 
-install(FILES mapviz_plugins.xml
-  DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION})
+install(FILES ${PROJECT_NAME}.xml
```

Comment:
The fact that this file has the same name as the package is incidental; maybe it should be hardcoded.